homeassistant/components/ecoal_boiler/__init__.py

```diff
@@ -0,0 +1,61 @@
+"""
+Support for ecoal/esterownik.pl coal/wood boiler controller.
+
+Allows read various readings available in controller
+and set very basic switches.
+"""
+import logging
+
+# import voluptuous as vol
+# import homeassistant.helpers.config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "ecoal_boiler"
+
+CONF_HOSTNAME = "host"
+CONF_LOGIN = "login"
+DEFAULT_LOGIN = "admin"
+CONF_PASSWORD = "password"
+DEFAULT_PASSWORD = "admin"
+
+
+# CONFIG_SCHEMA = vol.Schema({
+#    DOMAIN: vol.Schema({
+#        vol.Required(CONF_HOSTNAME): cv.string,
+#        vol.Optional(CONF_LOGIN,
+#                        default=DEFAULT_LOGIN): cv.string,
+#        vol.Optional(CONF_PASSWORD,
+#                        default=DEFAULT_PASSWORD): cv.string,
+#    })
+# })
```

Comment:
Configuration needs to be validated.

Reply:
I tried to validate by included but commented out code:
```
CONFIG_SCHEMA = vol.Schema({
   DOMAIN: vol.Schema({
       vol.Required(CONF_HOST): cv.string,
       vol.Optional(CONF_USERNAME,
                       default=DEFAULT_USERNAME): cv.string,
       vol.Optional(CONF_PASSWORD,
                       default=DEFAULT_PASSWORD): cv.string,
   })
})
```

But it fails with
`Invalid config for [ecoal_boiler]: [homeassistant] is an invalid option for [ecoal_boiler].`
:( for config:

```
ecoal_boiler:
  host: 192.168.9.2
  username: admin
  password: admin
```

I will really appreciate if someone enlightens me what I am doing wrong here.